DatabaseBuddy/ViewModel/MainWindowViewModel.cs

```
@@ -1031,17 +1244,24 @@ private void __GetMSSQLStudioPath(object obj = null)
             try
             {
                 var ExeName = "Ssms.exe";
-                foreach (string folder in Environment.GetEnvironmentVariable("path").Split(';'))
+                var Variables = Environment.GetEnvironmentVariable("path").Split(';').ToList();
+                Variables.Add(@"C:\Program Files (x86)");
+                foreach (string folder in Variables)
                 {
-                    if (File.Exists(folder + ExeName))
+                    try
                     {
-                        m_MSSQLStudioPath = folder + ExeName;
-                        break;
+                        var FilesInDirectory = Directory.GetFiles(folder, "*.exe", SearchOption.AllDirectories);
+                        var LocatedExe = FilesInDirectory.FirstOrDefault(x => x.EndsWith(ExeName, StringComparison.InvariantCultureIgnoreCase));
+                        if (LocatedExe.IsNotNullOrEmpty())
+                        {
+                            m_MSSQLStudioPath = LocatedExe;
+                            __WriteRegistryValue(nameof(m_MSSQLStudioPath), m_MSSQLStudioPath);
+                            break;
+                        }
                     }
-                    else if (File.Exists(folder + "\\" + ExeName))
+                    catch (Exception ex)
                     {
-                        m_MSSQLStudioPath = folder + "\\" + ExeName;
-                        break;
+
```

The reason for an empty catch block should be documented.